src/index.ts

```diff
@@ -155,6 +172,14 @@ export function withSpid(
       );
     })
     .map(spidStrategy => {
+      if (appConfig.startupIdpsMetadata) {
+        loadSpidStrategyOptions()
```

Comment:
this flow is not very clear. it looks like if `parse(startupIdpsMetadata)` fails we call `loadSpidStrategyOptions()`
the we call it again here (if there's a reason, can you add a comment to the code ?)

Reply:
Added a comment that explains this scenario. Fixed in 031a6f1